pkgs/by-name/j2/j2me/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, ant
+, jdk
+, stripJavaArchivesHook
+, SDL2
+, freeimage
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "j2me";
+  version = "0-unstable-2024-02-16";
+
+  src = fetchFromGitHub {
+    owner = "hex007";
+    repo = "freej2me";
+    rev = "8b9bc8a19baf26e3d92f88934a64a32f1cbc2795";
+    hash = "sha256-FaB2qnbtJnSZbrGRYDiz3Q/uNgD4GX//VaID6jHtyeU=";
+  };
+
+  nativeBuildInputs = [
+    ant
+    jdk
+    stripJavaArchivesHook
+  ];
+
+  buildInputs = [ SDL2 freeimage ];
```

Comment:
```
error: Package ‘freeimage-unstable-2021-11-01’ in /nix/store/x3751r06dj42dwwgdz03iir50j0929ff-source/pkgs/development/libraries/freeimage/default.nix:72 is marked as insecure, refusing to evaluate.


       Known issues:
        - CVE-2021-33367
        - CVE-2021-40262
        - CVE-2021-40263
        - CVE-2021-40264
        - CVE-2021-40265
        - CVE-2021-40266
        - CVE-2023-47992
        - CVE-2023-47993
        - CVE-2023-47994
        - CVE-2023-47995
        - CVE-2023-47996
```

Is this a concern that is managed in stable or should we do something about it?

Reply:
RFC NixOS Security Team: @risicle @LeSuisse @mweinelt

Reply:
Probably relevant: https://github.com/NixOS/nixpkgs/issues/298114

Reply:
I've commented on that issue, but just to repeat: freeimage is unmaintainable in a secure way. Upstream projects need to be told to stop depending on it.

Reply:
@risicle recommended alternative?

Heads-up: @hex007 (package repository owner)